.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.81.0
+    rev: v1.83.2
```

Comment:
Please upgrade this version to make sure that ordering in `wrappers` is ok - ref. https://github.com/antonbabenko/pre-commit-terraform/pull/565

Reply:
updated in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/305/commits/c016541dfe0e4249edcf5d358528853f78a3106e